.github/workflows/publish-to-test-pypi.yml

```diff
@@ -202,8 +202,21 @@ jobs:
     steps:
     - name: Install the package
       run: |
-        sleep 10
-        pip install --index-url https://pypi.org/simple/ --extra-index-url https://test.pypi.org/simple/ llama-stack==${{ needs.trigger-client-and-models-build.outputs.version }}
+        max_attempts=6
+        attempt=1
+        while [ $attempt -le $max_attempts ]; do
+          echo "Attempt $attempt of $max_attempts to install package..."
+          if pip install --index-url https://pypi.org/simple/ --extra-index-url https://test.pypi.org/simple/ llama-stack==${{ needs.trigger-client-and-models-build.outputs.version }}; then
```

Comment:
you can also do `--no-cache` which really forces pip to look harder. pip + pypi is not a good system unfortunately.

Reply:
done!